File: mycar.py
class mycar():
     def __init__(self, color,cost, type, min, max):
          self.color = color
          self.cost = cost
          self.type = type
          self.min = min
          self.max = max
     def getcolor(self):
          return self.color
     def getspeed(self):
          return self.max
     def getmin(self):
          return self.min
     def gettype(self):
          return self.type
     def getcost(self):
          return self.cost
     def print(self):
          return ("Hi! my car costs"+" "+ str(self.getcost()) + " " + "dollars" + "." +  "\n" + "its color is" +" "+ self.getcolor() +"." +"\n"
        + "it is a"+" "+ self.gettype() + " and it can speed up to" +" "+ str(self.getspeed()) +" "+ "mph" + "." + "\n")
BMW = mycar("twlight", 30000, "BMW", 0, 120)
print(BMW.print())

File: mycar.java
class mycar{
    int max_speed;
    int min_speed;
    String car_type;
    int car_cost;
    String color;
    public mycar(String color, int car_cost, String car_type, int min_speed, int max_speed ){
        this.color = color;
        this.car_cost = car_cost;
        this.car_type = car_type;
        this.min_speed = min_speed;
        this.max_speed = max_speed;
    }

    public String getcolor(){
        return color;
    }
    public int getspeed(){
        return max_speed;

    }
    public int getmin(){
        return min_speed; 
    }
    public String getcar_type(){
        return car_type;
    }
    public int getcost(){
        return car_cost;
    }
    public String print(){
        return ("Hi! my car costs"+" "+ this.getcost() + " " + "dollars" + "." +  "\n" + "its color is" +" "+ this.getcolor() +"." +"\n"
        + "it is a"+" "+ this.getcar_type() + " and it can speed up to" +" "+ this.getspeed() +" "+ "mph" + "." + "\n");
    }

    public static void main(String[] args){
    mycar BMW = new mycar("twilight", 30000, "BMW", 0, 120 );
    System.out.println(BMW.print());
    }

}